src/python/pants/jvm/jdk_rules_test.py

```diff
@@ -96,11 +96,8 @@ def test_java_binary_versions(rule_runner: RuleRunner) -> None:
     # default version is 1.11
     assert 'openjdk version "11.0' in run_javac_version(rule_runner)
 
-    rule_runner.set_options(["--jvm-tool-jdk=adopt:1.8"], env_inherit=PYTHON_BOOTSTRAP_ENV)
-    assert 'openjdk version "1.8' in run_javac_version(rule_runner)
-
-    rule_runner.set_options(["--jvm-tool-jdk=adopt:1.14"], env_inherit=PYTHON_BOOTSTRAP_ENV)
-    assert 'openjdk version "14"' in run_javac_version(rule_runner)
+    rule_runner.set_options(["--jvm-tool-jdk=temurin:1.17"], env_inherit=PYTHON_BOOTSTRAP_ENV)
```

Comment:
Note that this test is skipped due to flakiness, but I commented that out temporarily to verify that my changes work.